week2/week2-1/main.js

```diff
@@ -0,0 +1,118 @@
+import pic1 from "./assets/김규민.jpeg";
+import pic2 from "./assets/전희선.jpeg";
+import pic3 from "./assets/서혜은.jpg";
+import pic4 from "./assets/황주희.jpeg";
+import pic5 from "./assets/백지연.png";
+
+const $ = (selector) => document.querySelector(selector);
+
+let currentStep = 0;
+
+const quizList = [
+  {
+    src: pic1,
+    answer: "김규민",
+  },
+  {
+    src: pic2,
+    answer: "전희선",
+  },
+  {
+    src: pic3,
+    answer: "서혜은",
+  },
+  {
+    src: pic4,
+    answer: "황주희",
+  },
+  {
+    src: pic5,
+    answer: "백지연",
+  },
+];
+
+const score = $(".scoreBoard__score");
+const answer = $("ul.answer__list");
+const image = $(".imageBoard > img");
+
+window.onload = () => {
+  gameManager({
+    score: $(".scoreBoard__score"),
+    answer: $("ul.answer__list"),
+    image: $(".imageBoard > img"),
+  });
+};
+
```

Comment:
onload로 올려준거는 인자로 받아서 사용하면 되니까 굳이 다시 전역으로 선언하지 않아도 될 것 같아!
```suggestion
window.onload = () => {
  gameManager({
    score: $(".scoreBoard__score"),
    answer: $("ul.answer__list"),
    image: $(".imageBoard > img"),
  });
};

```

Reply:
이 부분도 ! gameManager 함수가 선언되고 그 뒤에서 window.onload를 해주는 편이 좀 더 좋을 것 같습니다!